### PR TITLE
107 urgent change address to major

### PR DESCRIPTION
**Pull request by belligerentbeagle**

### CODE REVIEW

**Review by ForAeons — CHANGES_REQUESTED** [review]
It seems that the example command is not updated (m/ADDRESS)

Screenshot 2024-04-04 at 7 16 57 PM

File: docs/DevOps.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ Any warnings or errors will be printed out to the console.
 
 Here are the steps to create a new release.
 
-1. Update the version number in [`MainApp.java`](https://github.com/AY2324S2-CS2103T-F13-1/tp/tree/master/src/main/java/seedu/address/MainApp.java).
+1. Update the version number in [`MainApp.java`](https://github.com/AY2324S2-CS2103T-F13-1/tp/tree/master/src/main/java/seedu/major/MainApp.java).
```
Comment (ForAeons) [inline docs/DevOps.md:79]: I believe this is a mistake, seedu.address is the package name. Ill use M to indicate similar errors

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ Given below is a quick overview of main components and how they interact with ea
 
 **Main components of the architecture**
 
-**`Main`** (consisting of classes [`Main`](https://github.com/AY2324S2-CS2103T-F13-1/tp/tree/master/src/main/java/seedu/address/Main.java) and [`MainApp`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/MainApp.java)) is in charge of the app launch and shut down.
+**`Main`** (consisting of classes [`Main`](https://github.com/AY2324S2-CS2103T-F13-1/tp/tree/master/src/main/java/seedu/major/Main.java) and [`MainApp`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/major/MainApp.java)) is in charge of the app launch and shut down.
```
Comment (ForAeons) [inline docs/DeveloperGuide.md:39]: I believe this is a mistake, seedu.address is the package name. Ill use M to indicate similar errors

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -69,13 +69,13 @@ The sections below give more details of each component.
 
 ### UI component
 
-The **API** of this component is specified in [`Ui.java`](https://github.com/AY2324S2-CS2103T-F13-1/tp/tree/master/src/main/java/seedu/address/ui/Ui.java)
+The **API** of this component is specified in [`Ui.java`](https://github.com/AY2324S2-CS2103T-F13-1/tp/tree/master/src/main/java/seedu/major/ui/Ui.java)
```
Comment (ForAeons) [inline docs/DeveloperGuide.md:72]: ditto

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -69,13 +69,13 @@ The sections below give more details of each component.
...
 
-The `UI` component uses the JavaFx UI framework. The layout of these UI parts are defined in matching `.fxml` files that are in the `src/main/resources/view` folder. For example, the layout of the [`MainWindow`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/ui/MainWindow.java) is specified in [`MainWindow.fxml`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/resources/view/MainWindow.fxml)
+The `UI` component uses the JavaFx UI framework. The layout of these UI parts are defined in matching `.fxml` files that are in the `src/main/resources/view` folder. For example, the layout of the [`MainWindow`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/major/ui/MainWindow.java) is specified in [`MainWindow.fxml`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/resources/view/MainWindow.fxml)
```
Comment (ForAeons) [inline docs/DeveloperGuide.md:78]: ditto

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -321,7 +321,7 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* * *`  | TA      | mark attendance for a student in my class | keep track of who's present |
 | `* * *`  | TA      | unmark attendance for a student in my class | keep track of who is absent |
 | `* * *`  | TA      | delete a student | remove a student if he or she leaves the class |
-| `* * *`  | TA      | know all the commands of the address book | use it effectively |
+| `* * *`  | TA      | know all the commands of the major book | use it effectively |
```
Comment (ForAeons) [inline docs/DeveloperGuide.md:324]: ditto

**Review by ForAeons — CHANGES_REQUESTED** [review]
Can help to look for seedu.major to address and reverse the change in all the documentation?

**Comment by tituschewxj** [comment]
Closed because of #145

**Comment by belligerentbeagle** [comment]
Closing as a branch of my PR was created by Titus to fix the accidental additional changes of address into major. 

#145

**Pull request closed without merging**